package.json

```diff
@@ -22,7 +22,7 @@
       "import": "./dist/index.js"
     }
   },
-  "main": "./dist/index.js",
+  "main": "./dist/index.cjs",
```

Comment:
I'm not sure about `type=module` package, should it point to CJS or ESM version?

Reply:
type=module's main should point to ESM

Reply:
Ok, just reverted.